vaadin-lumo-theme-flow-parent/vaadin-lumo-theme-flow/src/main/java/com/vaadin/flow/theme/lumo/LumoUtility.java

```diff
@@ -15,17 +15,15 @@
  */
 package com.vaadin.flow.theme.lumo;
 
-import com.vaadin.flow.component.dependency.JsModule;
-import com.vaadin.flow.component.dependency.NpmPackage;
-
 /**
  * Contains the definition for all the CSS utility classes provided by Lumo.
  */
-@NpmPackage(value = "@vaadin/vaadin-lumo-styles", version = "23.2.0-alpha2")
-@JsModule("@vaadin/vaadin-lumo-styles/utility.js")
-@JsModule("./lumo-utility.ts")
```

Comment:
@TatuLund Why have you removed these annotations?

Reply:
By accident, fixed